python/ray/data/aggregate.py

```diff
@@ -113,11 +113,16 @@ def __init__(self):
 class Sum(_AggregateOnKeyBase):
     """Defines sum aggregation."""
 
-    def __init__(self, on: Optional[KeyFn] = None, ignore_nulls: bool = True):
+    def __init__(self, on: Optional[KeyFn] = None, ignore_nulls: bool = True, alias_name: Optional[KeyFn] = None):
```

Comment:
Could you also add unit tests for this feature in test_dataset_all_to_all.py ?

Reply:
Yes, I will add test case in this file

Reply:
Thanks again, test case has been added to test_ dataset_ all_ to_ All files are included